examples/examples_nodejs_test.go

```diff
@@ -227,6 +227,24 @@ func TestAccMNG_withAwsAuth(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccMNG_DiskSize(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "tests", "managed-ng-disk-size"),
+			ExtraRuntimeValidation: func(t *testing.T, info integration.RuntimeValidationStackInfo) {
+				utils.RunEKSSmokeTest(t,
```

Comment:
They booted with the wrong disk size, so yeah, would be good to check. Is there another test I can look at that does similar verification, so I can use that as a basis?

Reply:
We could create a similar func like this one here that's checking if the node is ready: https://github.com/pulumi/pulumi-eks/blob/4e11388203c7e87032ac9074258bdb83a5ab3b85/examples/utils/utils.go#L371-L387

It could do something like:
```
var desiredSizeGB int64 = 50
	nodeEphemeralStorage := o.Status.Capacity[corev1.ResourceEphemeralStorage]
	return nodeEphemeralStorage.CmpInt64(desiredSizeGB * 1_000_000_000) >= 0
```

Alternatively we could check the EC2 instances directly using AWS APIs, but I feel like the kube only approach is cleaner.

Reply:
Just pushed a change to use the `utils.ValidateNodes` function. It's my first time using Go and I don't have a way to actually run the code so I'm sure it's missing something but hopefully it's a start?

Reply:
Thanks! I'll kick off the tests :)